media/css/mozorg/home/statement.scss

```diff
@@ -0,0 +1,223 @@
+    background-size: 20px 20px;
+    border-radius: 50%;
+    border: none;
+    display: none;
+    height: 35px;
+    padding: 6px;
+    position: absolute;
+    right: 15px;
+    top: 15px;
+    transition: background-color 0.2s ease;
+    width: 35px;
+
+    &:focus,
+    &:hover {
+        background-color: rgba(0, 0, 0, .3);
+        cursor: pointer;
+    }
+
+    // hide the 'Close' text
+    span {
+        @include visually-hidden;
+    }
+
+    // only display when JS is available
+    .js & {
+        display: block;
+    }
+}
+
+
+/*
+Animation
+============================================================================= */
+$delay: 0.2s;
+
+@mixin type-effect($paragraph-length) {
+    /* with graditiude to http://lea.verou.me/2012/02/simpler-css-typing-animation-with-the-ch-unit/*/
+    $cursor-speed: $paragraph-length; // characters per second
+    $duration: $paragraph-length / $cursor-speed * 1s;
+    $characters: $paragraph-length;
+
+    @keyframes letters-#{$paragraph-length} {
+        // animation, go from left to right
+        from {
+            left: 0;
+        }
+        to {
+            left: $paragraph-length + ch;
+        }
+    }
+
+    @keyframes highlight-#{$paragraph-length} {
+        // animation, go from left to right
+        from {
+            width: $paragraph-length + ch;
+        }
+        to {
+            width: 0;
+        }
+    }
+
+    max-width: $paragraph-length + ch;
+
+    &:before {
+        animation: highlight-#{$paragraph-length} 0.2s linear #{$delay * 2 + $duration} 1 both;
+    }
+
+    &:after {
+        animation: letters-#{$paragraph-length} $duration steps($paragraph-length, end) $delay 1 both;
+    }
+}
+
+@media #{$mq-desktop} {
+    @supports (width: 1ch) {
+        #promo-statement {
+            overflow: hidden;
+        }
+
+        .type-head span {
+            overflow: hidden;
+            padding-left: 0;
+            position: relative;
+            text-align: left;
+            word-break: break-all;
+
+            &:before {
+                background-color: #fff;
+                bottom: 0;
+                color: #000;
+                content: attr(data-content);
+                left: 0;
+                position: absolute;
+                top: 0;
+                width: 0;
+                overflow: hidden;
+                white-space: nowrap;
+            }
+
+            &:after {
+                background-color: #fff;
+                bottom: 0;
+                box-sizing: border-box;
+                content: '';
+                display: block;
+                position: absolute;
+                top: 0;
+                width: 100%;
+                z-index: 1;
+            }
+
+            .lang-en-US &,
+            .lang-en-GB & {
+                @include type-effect(32);
+            }
+
+            .lang-de & {
+                @include type-effect(44);
+            }
```

Comment:
Noting we'll need a `.lang-fr &` selector here...right?

Reply:
Yup. Still waiting on strings.